zathura-pdf-poppler/plugin.c

```diff
@@ -2,6 +2,8 @@
 
 #include "plugin.h"
 
+gboolean g_signature_overlay_toggle = true;
```

Comment:
If you need state, please keep it in inside zathura's document structure via `zathura_document_set_data`. As this currently stores the `PopplerDocument`, this would need to be refactored to store a custom struct with this variable and the `PopplerDocument` instance.